src/components/AddBook.js

```diff
@@ -25,6 +25,7 @@ function AddBook() {
 
   return (
     <>
+      <div className="line" />
       <div className="AddBook">
         <h2>ADD NEW BOOK</h2>
         <form className="bookform">
```

Comment:
- [ ] It would be better to change the input border color to the color in the Zephin design this will make your design look exactly like the Zephin design 👍🏿